core/src/test/java/novoda/lib/sqliteprovider/provider/SQLiteProviderLocalTest.java

```diff
@@ -212,10 +281,24 @@ private void insert(String uri, ContentValues cv) {
         provider.insert(Uri.parse("content://" + uri), cv);
     }
 
+    private void bulkInsert(String uri, ContentValues[] cv) {
+        provider.bulkInsert(Uri.parse("content://" + uri), cv);
+    }
+
     private void query(String uri) {
         provider.query(Uri.parse("content://" + uri), null, null, null, null);
     }
 
+    private ContentValues[] createContentValuesArray(int size) {
+        ContentValues[] bulkToInsert = new ContentValues[size];
+        ContentValues contentValues = new ContentValues();
+        contentValues.put("test", "test");
+        for (int i = 0; i < size; i++) {
+            bulkToInsert[i] = contentValues;
+        }
+        return bulkToInsert;
+    }
+
     public class SQLiteProviderImpl extends SQLiteContentProviderImpl {
```

Comment:
Not part of this PR, but just as a thought: this stub implementation could be easily replaced by a mock and allow use of `verify()` assertions along with `times()` and so forth.